backends/http.go

```diff
@@ -79,6 +80,11 @@ func NewHTTP(authOpts map[string]string, logLevel log.Level) (HTTP, error) {
 		missingOpts += " http_aclcheck_uri"
 	}
 
+	http.UserAgent = "mosquitto"
```

Comment:
Can we make the default a constant? Also, feel free not to export it, i.e. `userAgent` instead of `UserAgent`. I just haven't gotten to refactor `http` like I did with `jwt` backend.